guides/v2.1/security/security-overview.md

```diff
@@ -0,0 +1,20 @@
+---
+group: security
+title: Security
+functional_areas:
+  - Configuration
+---
+
+We have added security functionality for Magento Open Source, Commerce, and Commerce Cloud.  These modules enhance access security to protect your Magento instance from  unauthorized access, which can significantly reduce spam.
+
+## Google reCAPTCHA
+
+[Google reCAPTCHA]({{ page.baseurl }}/security/google-recaptcha.html) helps restrict access to the Magento Admin and store to humans only, not robots. It provides enhanced security over Magento CAPTCHA with additional benefits including Invisible reCAPTCHA.
+
+You will need to install the extension using Composer for v2.1 and v2.2. Installations and upgrades of v2.3 include reCAPTCHA.
+
+## Two-Faction Authentication
+
+[Two-Factor Authentication]({{ page.baseurl }}/security/two-factor-authentication.html)(2FA) adds support for software authentication apps and hardware authentication devices including Google Authenticator, Authy, U2F devices, and Duo Security. It provides additional security by requiring a second step to verify access through an entered code or push request.
+
+You will need to install the extension using composer for v2.1 and v2.2. Installations and upgrades of v2.3 include 2FA.
```

Comment:
You must install -- or you need to install ....

Reply:
Cutting some of this info, and revising.